app/jobs/google/search_keyword_job.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Google
+  class ClientServiceError < StandardError; end
```

Comment:
@ankitkalia1195 @junan 
This error relates to `ClientService` but is used by this job only (`SearchKeywordJob`). 
I think it is the right place to put it here, but I would appreciate any feedback or alternative~ 

Alternatives I though about:
- Bring it inside `ClientService` and update `ClientService#call` to raise an error instead of returning `false`. 
    I'm not happy with this as returning false makes the service cleaner and more usable.
- Calling an explicit `retry` instead of raising an error in the job.
    Jobs seems build to handle errors, there is no explicit `retry` method (to my knowledge) and calling a `perform_now` inside the `perform` method seems a bad workaround -- maybe I am wrong?
    
Thanks for your lights :)